.circleci/config.yml

```diff
@@ -115,6 +116,22 @@ jobs:
           path: /tmp/dist
           destination: cli/new-production
 
+  e2e-cli-node-12:
+    <<: *defaults
+    docker:
+      - image: *browsers_docker_image_node_12
+    environment:
+      BASH_ENV: ~/.profile
+      NPM_CONFIG_PREFIX: ~/.npm-global
+    resource_class: xlarge
+    parallelism: 4
+    steps:
+      - attach_workspace: *attach_options
+      - run: PATH=~/.npm-global/bin:$PATH xvfb-run -a node ./tests/legacy-cli/run_e2e --nb-shards=${CIRCLE_NODE_TOTAL} --shard=${CIRCLE_NODE_INDEX}
+      - store_artifacts:
```

Comment:
Remove this part so it doesn't overwrite the artifacts from the other job.

Reply:
Right you are, done.